src/main/java/xbot/common/logic/LogicUtils.java

```diff
@@ -0,0 +1,22 @@
+package xbot.common.logic;
+
+/**
+ * Utility methods for working with logic.
+ */
+public final class LogicUtils {
+    /**
+     * Returns true if any of the given values are true.
+     * @apiNote This is helpful if Checkstyle complains about boolean expression complexity.
+     * @param values The values to check.
+     *               If no values are given, this method will return false.
+     * @return True if any of the given values are true.
+     */
+    public static boolean anyOf(boolean... values) {
```

Comment:
That's crazy this isn't built in to Java, every other language seems to have it...

Reply:
should we add an `allOf` while we're here?

Reply:
Added.